packages/hardhat-truffle4/test/hardhat-project-with-test-contracts/contracts/fromContracts.sol

```diff
@@ -0,0 +1 @@
+contract A {}
```

Comment:
Compilation fails if there's a single, empty file in the contracts directory. It works fine if the directory is empty. It also fails if the file only has, for example, a struct (but no contracts).

This seems like a bug, albeit not a very important one IMO.

Reply:
Fixed it